stablehlo/integrations/python/StablehloModule.cpp

```diff
@@ -474,4 +476,39 @@ PYBIND11_MODULE(_stablehlo, m) {
                                        stablehloTypeExtensionsGetBoundsSize,
                                        stablehloTypeExtensionsGetBoundsElem);
       });
+
+  //
+  // Serialization APIs.
+  //
+
+  m.def("get_current_version",
+        []() { return mlir::stablehlo::getCurrentVersion(); });
```

Comment:
Let's align the order of these functions between Serialization.h and StablehloModule.cpp? I don't have a preference for the order, just for it being the same across files.